dbld/rules

```diff
@@ -19,6 +19,7 @@ RELEASE_DIR=$(DBLD_DIR)/release
 VERSION=$(shell cat VERSION)
 TARBALL=$(BUILD_DIR)/syslog-ng-$(VERSION).tar.gz
 MODE=snapshot
+CONFIGURE_OPTS=--enable-debug --enable-manpages --with-python=3 --prefix=/install --with-docbook=/usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl $(CONFIGURE_ADD)
```

Comment:
In the admin guide we state: Currently only Python 2.7 is supported. Let's align --with-python=2 with the documentation.

Reply:
We can do that in the initial merging, however I would like us to start testing Python3 in Jenkins and possibly change the documentation later. We used to work with Python3 and then broke it. This was the state that got documented.

But I'll change it for now.

Reply:
If we want jenkins to use this line, then this should be parametrized somehow. As we do not want to drop support for v2, jenkins should do both.

Reply:
You can parametrize it using the CONFIGURE_ADD variable that you can pass via the make command line or even as environment variable.

Or you can override CONFIGURE_OPTS as a whole.

make -f dbld/rules CONFIGURE_OPTS=whatever

Reply:
Ok fine for me. Thanks for explaining. I have no other comments.